docs/topics/native/native-cocoapods.md

```diff
@@ -12,26 +12,35 @@ To work correctly with Xcode, you should [update your Podfile](#update-podfile-f
 Depending on your project and purposes, you can add dependencies between [a Kotlin project and a Pod library](native-cocoapods-libraries.md)
 as well as [a Kotlin Gradle project and an Xcode project](native-cocoapods-xcode.md).
 
-## Set up the environment to work with CocoaPods
+## Set up an environment to work with CocoaPods
 
-Install the [CocoaPods dependency manager](https://cocoapods.org/):
+1. Install the [CocoaPods dependency manager](https://cocoapods.org/):
 
-```ruby
-sudo gem install cocoapods
-```
+   * In the IDE terminal or your command-line tool, run the following command to install CocoaPods using Homebrew:
+
+       ```Shell
+       brew install cocoapods
+       ```
+
+   * If you don't have Homebrew yet, [install it](https://brew.sh/) or use the default Ruby available on macOS:
+
+       ```ruby
+       sudo gem install cocoapods
+       ```
 
-* If you use Kotlin prior to version 1.7.0, install the [`cocoapods-generate`](https://github.com/square/cocoapods-generate)
-plugin:
+2. If you use Kotlin prior to version 1.7.0, you also need the [`cocoapods-generate`](https://github.com/square/cocoapods-generate)
```

Comment:
Better to hide this text because some users can copy-paste commands with no attention on the explanations

Reply:
Good idea, moved it to a collapsible block